internal/service/handlers/registration.go

```diff
@@ -41,6 +41,17 @@ func Registration(w http.ResponseWriter, r *http.Request) {
 	})
 	log.Debug("registration request")
 
+	registrationAddress := RelayerConfig(r).RegistrationAddress
```

Comment:
we should know that this is default. Consider renaming or adding comment somewhere

Reply:
Comment added